source/topics/paas/how-to-setup-application-monitoring.md

```diff
@@ -0,0 +1,9 @@
+Aptible recommends New Relic for application monitoring, and now supports it at the [host-level](https://github.com/aptible/support/issues/115) as well.
```

Comment:
Let's consistently use "Server Monitoring" and "Application Monitoring (APM)" [sic] when referring to the two services, instead of "host-level", "application monitoring", etc.